h/browser/chrome/lib/tab-state.js

```diff
@@ -151,31 +148,13 @@ function TabState(initialState, onchange) {
    * @param {string} apiUrl The URL of the Hypothesis API.
    */
   this.updateAnnotationCount = function(tabId, tabUrl, apiUrl) {
```

Comment:
`apiUrl` is now an unused parameter -- you need to update the function signature and the docstring.